.eslintrc.js

```diff
@@ -475,7 +475,7 @@ module.exports = {
       rules: {
         'import/no-default-export': 'error',
         'import/prefer-default-export': 'off',
-        ...(ENABLE_REACT_COMPILER_PLUGIN ? { 'react-compiler/react-compiler': 'off' } : {}),
+        'react-compiler/react-compiler': 'off',
```

Comment:
Is `ENABLE_REACT_COMPILER_PLUGIN` no longer useful?

Reply:
For now we leave it always off for Base UI.